packages/hurl/src/cli/options/commands.rs

```diff
@@ -561,6 +561,27 @@ pub fn test() -> clap::Arg {
         .action(clap::ArgAction::SetTrue)
 }
 
+pub fn test_retry() -> clap::Arg {
+    clap::Arg::new("test_retry")
+        .long("test-retry")
+        .value_name("NUM")
+        .value_parser(clap::value_parser!(u32).range(0..))
+        .allow_hyphen_values(true)
+        .help("Maximum number of test retries, 0 for no retries")
+        .help_heading("Run options")
+        .num_args(1)
+}
+
+pub fn test_retry_interval() -> clap::Arg {
+    clap::Arg::new("test_retry_interval")
+        .long("test-retry-interval")
+        .value_name("MILLISECONDS")
+        .default_value("0")
+        .help("Interval in milliseconds before a retrying a test")
+        .help_heading("Run options")
+        .num_args(1)
+}
+
```

Comment:
We should probably call this something other than `test-retry` beacuse it has nothing to do with `--test` really.